cmd/collectors/zapi/plugins/qtree/qtree.go

```diff
@@ -281,5 +279,10 @@ func (my *Qtree) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 		Str("parseD", parseT.Round(time.Millisecond).String()).
 		Str("batchSize", my.batchSize).
 		Msg("Collected")
-	return []*matrix.Matrix{my.data}, nil
+
+	// This would generate quota metrics prefix with `qtree_`. These are deprecated now and will be removed later.
+	qtreePluginData := collectors.CloneWithOtherIdentifier(my.Parent+".Qtree", "qtree", "qtree", my.data)
+	output = append(output, qtreePluginData)
```

Comment:
minor: these lines can be replaced with `	return []*matrix.Matrix{qtreePluginData, my.data}, nil`